reversion/cores/views.py

```diff
@@ -8,6 +8,34 @@
 from is_core.generic_views.form_views import EditModelFormView, DetailModelFormView
 from is_core.patterns import reverse_pattern
 
+from reversion.models import Version
+
+
+class VersionInlineFormView(TabularInlineObjectsView):
+    model = Version
+    fields = (
+        ('type', _('type')),
+        ('content_type', _('content type')),
+        ('object', _('object')),
+        ('data', _('data')),
+    )
+
+    def parse_object(self, obj):
+        return {'type': obj.get_type_display(), 'object': self.get_object(obj), 'content_type': obj.content_type,
+                'data': ', '.join(('%s: %s' % (k, v if v != '' else '--') for k, v in obj.flat_field_dict.items()))}
```

Comment:
možná bych dal klíče na nové řádky, hodnoty jsou celkem dlouhé, ale je to kosmetika...